BridgeSDK/BridgeAPI/SBBActivityManager.h

```diff
@@ -59,26 +59,35 @@ typedef void (^SBBActivityManagerUpdateCompletionBlock)(id responseObject, NSErr
 /*!
  Gets all available, started, or scheduled activities for a user. The "daysAhead" parameter allows you to retrieve activities that are scheduled in the future for the indicated number of days past today (to a maximum of four days, at this time). This allows certain kinds of UIs (e.g. "You have N activities tomorrow" or "You have completed N of X activities today", even when the activities are not yet to be performed). A "daysBehind" parameter allows you to retain previously-cached activities that have not yet been completed that expired within the indicated number of days in the past. This allows UIs that say, e.g., "You left N activities uncompleted yesterday." Scheduled activities will be returned in the timezone of the device at the time of the request. Once a task is finished, or expires (the time has passed for it to be started), or becomes invalid due to a schedule change on the server, it will be removed from the list of scheduled activities returned from Bridge, and (except for previously-fetched but unfinished tasks within daysBehind) will also be removed from the list passed to this method's completion handler.
  
+ If activities are in the process of being updated already, because either this or another method that can update the cached scheduled activities was called in another thread and the completion handler hasn't returned, this method will either block until available, or return nil and the completion handler will be called with an SBBActivitiesBeingUpdatedError error, depending on the value passed in to the wait parameter.
+ 
  @param daysAhead  A number of days in the future (0-4) for which to retrieve available/started/scheduled activities.
  @param daysBehind  A number of days in the past (no limit) for which to include previously-cached but expired and unfinished activities (ignored if the SDK was initialized with useCache=NO).
  @param policy Caching policy to use (ignored if the SDK was initialized with useCache=NO).
+ @param wait Whether to wait for any previous activity updates to complete, or to just exit if busy.
```

Comment:
What is your thinking for when the app wouldn't want to wait? It's an asynchronous design already. Is this to handle the case were the user flips back and forth from activities and it hasn't returned results before firing off next "go update my activities" call?